config/templates/folio_to_hyacinth.yml

```diff
@@ -1,5 +1,11 @@
 development:
   download_directory: <%= Rails.root.join('tmp/working_data/development/folio_to_hyacinth/downloaded_files') %>
+  project_mappings:
```

Comment:
I think you can remove this project_mappings configuration option.  I don't think it's necessary, and I'll say more in a later comment in this PR (where it's currently being used).